doc/gemini_gmos.rst

```diff
@@ -10,6 +10,14 @@ This file summarizes several instrument specific
 settings that are related to the Gemini/GMOS spectrograph.
 
 
+Nod and Shuffle
+===============
+
+For the time being, we have enabled reductions of data
+taken in Nod+Shuffle mode by simply replicating the calibrations.
+That is, we do *not* subtract the nodded images but reduce
+it as if it were a separate slit.
+
```

Comment:
I guess this is only for my understanding. In this case can the users subtract the nodded reduced 2D spectra themselves, or will they only be able to coadd the 1D spectra extracted from each frame?

Reply:
The latter, primarily.  Although I guess they'll have the 2D reduced
frames that they could subtract as well.